go.mod

```diff
@@ -3,30 +3,29 @@ module github.com/mattermost/calls-transcriber
 go 1.21.4
 
 require (
-	github.com/mattermost/mattermost-plugin-calls/server/public v0.0.3
-	github.com/mattermost/mattermost/server/public v0.0.6
-	github.com/mattermost/rtcd v0.12.1-0.20240105175112-64f91e1de8e0
+	github.com/mattermost/mattermost-plugin-calls/server/public v0.0.0-20240308191258-3efb429339df
+	github.com/mattermost/mattermost/server/public v0.0.12
+	github.com/mattermost/rtcd v0.14.0-dev.0.20240314152404-efacb91f3c4f
```

Comment:
We should update this to point to master.

Reply:
Can we cut a new version and point to that?

Reply:
Yes, it's practically the same though :)

Reply:
https://github.com/mattermost/rtcd/tree/v0.14.0